requirements/extras/sqs.txt

```diff
@@ -1,3 +1,3 @@
-boto3>=1.4.4
-pycurl==7.43.0.2  # Latest build with wheels provided
-urllib3<1.26      # Unittests are faiing with urllib3>=1.28
+boto3>=1.9.12
+pycurl~=7.44.1  # Latest build with wheels provided
+urllib3>=1.26.7    # Unittests are faiing with urllib3>=1.28
```

Comment:
Is the comment here still true regarding issues with `urllib3>=1.28`? If so should there still be an upper bound on this (if not the comment can be removed)

Reply:
https://pypi.org/project/urllib3/#history says there were no 1.28 and its a typo. and should be 1.26. if you do no use windows, you should be pretty safe to upgrade as pycurl latest dont provide any wheel in pypi. you/we have to generate pycurl wheel for windows.